### PR TITLE
Fix: Upgrade to react-scripts@3.1.1

### PR DESCRIPTION
**Pull request by Boomatang**
## Motivation
https://issues.jboss.org/browse/AGMS-925

## What
Upgrade to react-scripts@3.1.1

## Why
Fix possible security issues

## Verification Steps
1. Remove node_modules
2. Run `npm install`
3. Run `npm run test` & `npm run lint`
4. Manually check site works

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Additional Notes
- you may be required to remove the package-lock.json file if there is issues running the eslint

PS.: Add images and/or .gifs to illustrate what was changed if this pull request modifies the appearance/output of something presented to the users.

### CODE REVIEW

**Comment by ziccardi** [comment]
👀

File: src/components/common/BuildStatus.test.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ describe('BuildStatus', () => {
       Running: '.fa-refresh',
       Succeeded: '.text-success'
     };
+    // eslint-disable-next-line no-unused-vars
```
Comment (ziccardi) [inline src/components/common/BuildStatus.test.js]: @Boomatang Why do we disable the `no-unused-vars` check?

Comment (Boomatang) [reply src/components/common/BuildStatus.test.js]: There was an issue with eslint saying there was usnused-vars in the for loops. We have found a fix. babel-eslint@10.0.3 needs to be add. I am doing the update now.

File: src/components/mobileservices/validator/RuleSetValidator.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ export class RuleSetValidator {
       return false;
     }
     let hasErrors = false;
+    // eslint-disable-next-line no-unused-vars
```
Comment (ziccardi) [inline src/components/mobileservices/validator/RuleSetValidator.js]: Do we still need this?

Comment (Boomatang) [reply src/components/mobileservices/validator/RuleSetValidator.js]: Nope, We do not. I missed these two the first time around. They should be changed now.

File: package.json [hunk 3]
```diff
@@ -32,7 +32,7 @@
     "react-moment": "0.9.2",
     "react-redux": "5.1.1",
     "react-router-dom": "5.0.1",
-    "react-scripts": "3.0.1",
+    "react-scripts": "^3.1.1",
```
Comment (craicoverflow) [inline package.json]: Version should be fixed.

```suggestion
"react-scripts": "3.1.1",
```

File: package.json [hunk 4]
```diff
@@ -55,6 +55,7 @@
   },
   "homepage": "/",
   "devDependencies": {
+    "babel-eslint": "^10.0.3",
```
Comment (craicoverflow) [inline package.json]: Version should be fixed.

```suggestion
"babel-eslint": "10.0.3",
```

**Comment by craicoverflow** [comment]
> babel-eslint@10.0.3

babel-eslint@10.0.3 had to be installed to fix https://github.com/babel/babel-eslint/issues/791

I'm not sure what else we could do.

**Pull request merged**